.changeset/soft-stars-pay.md

```diff
@@ -0,0 +1,8 @@
+---
+'@graphql-tools/load': minor
```

Comment:
This feels like a `minor`, rather than a `patch` because it's not a bug per se
It's _could_ be a `major` if consumers specifically check `Error` case 🤔 

Happy to be swayed either way here

Reply:
It is still Error right? So it should be good imo.